kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "opendistroAnomalyDetectionKibana",
+  "version": "1.11.0.0",
+  "kibanaVersion": "7.9.1",
+  "configPath": ["opendistro_anomaly_detection_kibana"],
+  "requiredPlugins": ["navigation"],
+  "optionalPlugins": ["opendistro-alerting", "opendistro_alerting"],
```

Comment:
Why add two alerting plugins?

Reply:
This was the naming conventions for the ES and Kibana alerting plugins. With the new naming conventions for the Kibana plugin this will need to be revisited, I'll make a note of it. Also, adding optional plugins doesn't really have much functionality at all, and we could remove altogether. See [here](https://github.com/elastic/kibana/blob/master/docs/development/core/server/kibana-plugin-core-server.pluginmanifest.md)